sr_robot_launch/launch/sr_bimanual_simulation_control_loop.launch

```diff
@@ -44,6 +44,7 @@
 
   <!-- Robot description related config-->
   <arg name="robot_model" default="ur10e"/>
+  <arg name="ignore_robot_model" default="" />
```

Comment:
is that a boolean? should we set a proper default value?

Reply:
no its the robot name to ignore. https://github.com/shadow-robot/sr_interface/blob/noetic-devel/sr_robot_launch/launch/sr_bimanual_simulation_control_loop.launch#L119